packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -185,6 +188,31 @@ export const VSelect = genericComponent<new <
       } else if (e.key === 'End') {
         listRef.value?.focus('last')
       }
+
+      // html select hotkeys
+      ((e: KeyboardEvent) => {
+        const KEYBOARD_LOOKUP_THRESHOLD = 1000 // milliseconds
+
+        function checkPrintable (e: KeyboardEvent) {
+          const isPrintableChar = e.key.length === 1 && e.key !== ' '
+          const noModifier = !e.ctrlKey && !e.metaKey && !e.altKey
+          return isPrintableChar && noModifier
+        }
+
+        if (props.multiple || !checkPrintable(e)) return
```

Comment:
- We want to keep spaces
- Not sure why you'd use an iife here
```suggestion
      const KEYBOARD_LOOKUP_THRESHOLD = 1000 // milliseconds
      const isPrintable = e.key.length === 1 && !e.ctrlKey && !e.metaKey && !e.altKey
      if (isPrintable && !props.multiple) {
```

Reply:
Not particular good reason, I was just paranoid at that moment wanting to group this logic in a dedicated place